modules/nux/site.js

```diff
@@ -53,11 +53,11 @@ var display_final_nux_step = function(res) {
                 {'name': 'imap_server_id', value: res.nux_server_id},
                 {'name': 'imap_service_name', value: res.nux_service_name}],
                 function () {
-                    Hm_Utils.redirect();
+                    Hm_Utils.redirect('?page=servers');
```

Comment:
Why do you redirect to a specific page? Isn't the nux module also having a separate url that uses the server forms? redirect without any argument should actually redirect to the same page.